focus-android/app/src/main/java/org/mozilla/focus/settings/SettingsFragment.kt

```diff
@@ -51,41 +42,6 @@ class SettingsFragment : BaseSettingsFragment() {
         return super.onPreferenceTreeClick(preference)
     }
 
-    override fun onCreateMenu(menu: Menu, menuInflater: MenuInflater) {
-        menuInflater.inflate(R.menu.menu_settings_main, menu)
-    }
-
-    override fun onMenuItemSelected(menuItem: MenuItem): Boolean {
-        if (menuItem.itemId == R.id.menu_whats_new) {
-            whatsNewClicked()
-            return true
-        }
-        return false
-    }
-
-    private fun whatsNewClicked() {
-        val context = requireContext()
-
-        SettingsScreen.whatsNewTapped.add()
-
-        TelemetryWrapper.openWhatsNewEvent(WhatsNew.shouldHighlightWhatsNew(context))
-
-        WhatsNew.userViewedWhatsNew(context)
-
-        val sumoTopic = if (AppConstants.isKlarBuild) {
-            SupportUtils.SumoTopic.WHATS_NEW_KLAR
-        } else {
-            SupportUtils.SumoTopic.WHATS_NEW_FOCUS
-        }
-
-        val url = SupportUtils.getSumoURLForTopic(SupportUtils.getAppVersion(context), sumoTopic)
```

Comment:
This `getSumoURLForTopic` function can also be removed too.

Reply:
Why does this function need to be removed? It's being utilized at other places too.

Reply:
Oops, you're right. 🙂